pom.xml

```diff
@@ -173,6 +152,82 @@
                 <additionalparam>-Xdoclint:none</additionalparam>
             </properties>
         </profile>
+        <profile>
+            <id>sonar</id>
```

Comment:
We should also remove https://github.com/symphonyoss/messageml-utils/settings/hooks/278953455

Reply:
I was thinking to leave it as it works for external PR (Sonar is limited to internal PR, not from a fork, because it needs to access the Sonar token)